.gitignore

```diff
@@ -1,3 +1,5 @@
 /docker-compose.override.yaml
 /docker-compose.override.yml
 /flarectl*
+/api/src/Controller/TestsController.php
```

Comment:
I don't see how this file could exist, so why adding it in this gitignore file? (Same about Makefile)

If those files only exist on your machine, please just ignore them on commit, but not in the project configuration files